Tests are not passing

Tests are not passing locally. On current master. This is the traceback:

```
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.7.3, pytest-3.10.1, py-1.8.0, pluggy-0.12.0
rootdir: /home/martin/github/martinRenou/voila, inifile:
plugins: tornado5-2.0.0
collected 20 items                                                                                                                                                                           

tests/execute_output_test.py .                                                                                                                                                         [  5%]
tests/app/config_paths_test.py ....                                                                                                                                                    [ 25%]
tests/app/cwd_subdir_test.py .                                                                                                                                                         [ 30%]
tests/app/cwd_test.py .                                                                                                                                                                [ 35%]
tests/app/execute_test.py ..                                                                                                                                                           [ 45%]
tests/app/image_inlining_test.py .                                                                                                                                                     [ 50%]
tests/app/nbextensions_test.py .                                                                                                                                                       [ 55%]
tests/app/no_strip_sources_test.py .                                                                                                                                                   [ 60%]
tests/app/serve_directory_test.py .                                                                                                                                                    [ 65%]
tests/app/template_cli_test.py .                                                                                                                                                       [ 70%]
tests/app/template_custom_test.py .                                                                                                                                                    [ 75%]
tests/app/template_gridstack_test.py .                                                                                                                                                 [ 80%]
tests/server/cwd_subdir_test.py F                                                                                                                                                      [ 85%]
tests/server/execute_test.py F                                                                                                                                                         [ 90%]
tests/server/nbextensions_test.py x                                                                                                                                                    [ 95%]
tests/server/no_strip_sources_test.py F                                                                                                                                                [100%]

========================================================================================== FAILURES ==========================================================================================
______________________________________________________________________________________ test_hello_world ______________________________________________________________________________________

http_client = <tornado.simple_httpclient.SimpleAsyncHTTPClient object at 0x7f4fbe719518>, cwd_subdir_notebook_url = 'http://localhost:45383/voila/render/subdir/cwd_subdir.ipynb'

    @pytest.mark.gen_test
    def test_hello_world(http_client, cwd_subdir_notebook_url):
>       response = yield http_client.fetch(cwd_subdir_notebook_url)

tests/server/cwd_subdir_test.py:15: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tornado.gen.Runner object at 0x7f4fbc436240>

    def run(self) -> None:
        """Starts or resumes the generator, running until it reaches a
        yield point that is not ready.
        """
        if self.running or self.finished:
            return
        try:
            self.running = True
            while True:
                future = self.future
                if future is None:
                    raise Exception("No pending future")
                if not future.done():
                    return
                self.future = None
                try:
                    exc_info = None
    
                    try:
>                       value = future.result()
E                       tornado.httpclient.HTTPClientError: HTTP 404: Not Found

../../../miniconda3/envs/voila/lib/python3.7/site-packages/tornado/gen.py:729: HTTPClientError
----------------------------------------------------------------------------------- Captured stderr setup ------------------------------------------------------------------------------------
[W 09:37:10.774 ServerApp] All authentication is disabled.  Anyone who can connect to this server will be able to run code.
------------------------------------------------------------------------------------ Captured stderr call ------------------------------------------------------------------------------------
[E 09:37:10.794 ServerApp] Could not open static file ''
[W 09:37:10.794 ServerApp] 404 GET /voila/render/subdir/cwd_subdir.ipynb (127.0.0.1) 14.89ms referer=None
______________________________________________________________________________________ test_hello_world ______________________________________________________________________________________

http_client = <tornado.simple_httpclient.SimpleAsyncHTTPClient object at 0x7f4fbc3d2358>, print_notebook_url = 'http://localhost:41907/voila/render/print.ipynb'

    @pytest.mark.gen_test
    def test_hello_world(http_client, print_notebook_url):
>       response = yield http_client.fetch(print_notebook_url)

tests/server/execute_test.py:16: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tornado.gen.Runner object at 0x7f4fbc3d2ef0>

    def run(self) -> None:
        """Starts or resumes the generator, running until it reaches a
        yield point that is not ready.
        """
        if self.running or self.finished:
            return
        try:
            self.running = True
            while True:
                future = self.future
                if future is None:
                    raise Exception("No pending future")
                if not future.done():
                    return
                self.future = None
                try:
                    exc_info = None
    
                    try:
>                       value = future.result()
E                       tornado.httpclient.HTTPClientError: HTTP 404: Not Found

../../../miniconda3/envs/voila/lib/python3.7/site-packages/tornado/gen.py:729: HTTPClientError
----------------------------------------------------------------------------------- Captured stderr setup ------------------------------------------------------------------------------------
[W 09:37:10.861 ServerApp] All authentication is disabled.  Anyone who can connect to this server will be able to run code.
------------------------------------------------------------------------------------ Captured stderr call ------------------------------------------------------------------------------------
[W 09:37:10.880 ServerApp] 404 GET /voila/render/print.ipynb (127.0.0.1) 14.99ms referer=None
______________________________________________________________________________________ test_hello_world ______________________________________________________________________________________

http_client = <tornado.simple_httpclient.SimpleAsyncHTTPClient object at 0x7f4fbc453eb8>, print_notebook_url = 'http://localhost:42023/voila/render/print.ipynb'

    @pytest.mark.gen_test
    def test_hello_world(http_client, print_notebook_url):
>       response = yield http_client.fetch(print_notebook_url)

tests/server/no_strip_sources_test.py:16: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tornado.gen.Runner object at 0x7f4fbc3c6f98>

    def run(self) -> None:
        """Starts or resumes the generator, running until it reaches a
        yield point that is not ready.
        """
        if self.running or self.finished:
            return
        try:
            self.running = True
            while True:
                future = self.future
                if future is None:
                    raise Exception("No pending future")
                if not future.done():
                    return
                self.future = None
                try:
                    exc_info = None
    
                    try:
>                       value = future.result()
E                       tornado.httpclient.HTTPClientError: HTTP 404: Not Found

../../../miniconda3/envs/voila/lib/python3.7/site-packages/tornado/gen.py:729: HTTPClientError
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -98,19 +98,30 @@ voila
 
 ## Tests
 
-To run the tests:
+Install the test dependencies
 
 ```bash
-python -m pip install ".[test]"
+python -m pip install -e ".[test]"
+```
+
+Enable the Jupyter server extension:
+
+```bash
+jupyter extension enable voila --sys-prefix
+```
+
+Running the tests locally also requires the `test_template` to be installed:
+
+```bash
+python -m pip install ./tests/test_template
+```
+
+Finally, to run the tests:
+
+```bash
 python -m pytest
 ```
 
-Running the tests locally requires the `test_template` to be installed:
-
-```bash
-cd tests/test_template
-python -m pip install .
-```
 
 ## Editing templates
 

